en/docs/learn/data-publishing-reports.md

```diff
@@ -176,4 +198,4 @@ following reports according to the OBIE templates:
 - Additional Metrics - Revocation Notifications from ASPSPs
 - Additional Metrics - AIS Corporate Account Access (OPTIONAL)
 - Additional Metrics - Payment Refunds
-- Additional Metrics - 90 Days Re-authentication
\ No newline at end of file
+- Additional Metrics - 90 Days Re-authentication_
```

Comment:
Is the underscore here necessary?  `Additional Metrics - 90 Days Re-authentication_`

Reply:
Fixed in fae1a3d4d37ff71f5893786f015c13654abdc2d9.